grails-app/init/de/iteratec/osm/FrontendWatcher.groovy

```diff
@@ -28,6 +28,10 @@ class FrontendWatcher {
         String nodeLocation = new File(nodeExecLocation).getParent()
 
         if (location && nodeLocation) {
+
+            String frontendDistFolder = Paths.get("${location}/frontend/dist")
+            new File(frontendDistFolder).mkdirs()
```

Comment:
Did you check whether this fails if the folder already exists?

Reply:
.mkdirs() just returns false and does nothing if the folder or parent folders does not exist.

Reply:
I think you mean if it DOES exist.